src/index.ts

```diff
@@ -38,19 +45,20 @@ export default class ApiClient {
     summary,
     content,
     tags,
+    files,
   }: Post): Promise<Response<Post>> {
+    let formData = new FormData();
+    formData.append("title", title);
+    if (summary) {
+      formData.append("summary", summary);
+    }
```

Comment:
Is summary required or not, I thought your changes to the backend made it required?

Reply:
Yes, it is mandatory. I was just mindlessly rewriting the original code there.